tests/e2e/asserts_test.go

```diff
@@ -178,6 +178,47 @@ func AssertSwitchoverWithHistory(
 				return nil
 			}, timeout).ShouldNot(HaveOccurred())
 		})
+	} else {
+		By("verifying that the all standbys are streaming from the new primary", func() {
+			timeout := 120
+			commandTimeout := time.Second * 10
+			Eventually(func(g Gomega) {
+				podList, err := env.GetClusterPodList(namespace, clusterName)
+				g.Expect(err).ToNot(HaveOccurred())
+				g.Expect(podList.Items).To(HaveLen(oldPodListLength))
+				var standbys []interface{}
```

Comment:
I believe this is a `[]string`

Reply:
This adheres to the interface of `ContainElements`

https://github.com/onsi/gomega/blob/master/matchers.go#L420-L424

Reply:
oh, the `ContainElements` flattens the slice internally, please ignore the above comment 

https://github.com/onsi/gomega/blob/master/matchers/consist_of.go#L62-L72